drivers/entropy/entropy_stm32.c

```diff
@@ -88,8 +86,7 @@ struct entropy_stm32_rng_dev_data {
 };
 
 static const struct entropy_stm32_rng_dev_cfg entropy_stm32_rng_config = {
-	.pclken	= { .bus = DT_INST_CLOCKS_CELL(0, bus),
-		    .enr = DT_INST_CLOCKS_CELL(0, bits) },
+	.pclken = { .bus = DT_INST_CLOCKS_CELL(0, bus), .enr = DT_INST_CLOCKS_CELL(0, bits) },
```

Comment:
yeah, it doesn't add up to the readability indeed..
I will revert the clang format commit since it's more question to debate than it helps it seems :-).

I'm really dissapointed (as you are probably) in the benefit of the current clang configuration as it is never helping to pass a review for code style. I think it's a real pain for new developers and reviewers especially since both lose time on something which can be done more automatically. Is there any issue for this, or should I/you create one?

Reply:
reverted

Reply:
I agree, Zephyr should take this more seriously. In year 2022 we should no longer be discussing code formatting but rather say “please run this auto-formatter”. It's an area that IMO is crucial for a project to scale well. I've been working recently on simplifying `clang-format`, see https://github.com/zephyrproject-rtos/zephyr/pull/41307.